src/rad/handler.clj

```diff
@@ -138,11 +149,13 @@
       :description (:description page)
       :level level
     }
-  )
-)
+  ))
+
+(defn ms-from-epoch []
+  (.getTime (new java.util.Date)))
 
 (defroutes app-routes
-  (GET "/api" [] (json/write-str (select-page pages)))
+  (GET "/api" [] (json/write-str (select-page (ms-from-epoch) (pages))))
```

Comment:
I want to keep the logic in the server. A few reasons:

- I want to keep the frontend super thin and dummy. It just polls and renders what it gets. That's it.
- I want that all frontend clients would show the same page at the same time
- There's now some additional logic added to the `select-page` function. It puts problems and warnings to higher priority. I don't want to move that logic to the frontend.